ports/mimxrt/Makefile

```diff
@@ -209,6 +210,11 @@ ifeq ($(MICROPY_PY_MACHINE_SDCARD),1)
 SRC_HAL_IMX_C += $(MCU_DIR)/drivers/fsl_usdhc.c
 endif
 
+ifneq ($(MICROPY_HW_QTMR_SUPPORT),0)
```

Comment:
It might be cleaner to do something like the following (which is what stm32 does in these cases):
```
ifeq ($(MCU_SERIES),$(filter $(MCU_SERIES),MIMXRT1015 MIMXRT1021 MIMXRT1052 MIMXRT1062))
...
endif
```

Reply:
Since there are not too many PR's open at the moment, I would prefer adding MICROPY_HW_QTMR_SUPPORT=1 to the config files of the respective boards. That avoids a hiccup if another MCU type is added. Unfortunately one cannot extract easily a setting from a header file into the Makefile. Otherwise the information is in the MCU-Specific feature header file.

Reply:
Yes, sounds good (to add `MICROPY_HW_QTMR_SUPPORT=1` where applicable).

Reply:
I changed my mind into following your suggestion. There is always more than one way to forget things, and it does not make a difference where to change if another MCU is added.
b.t.w.: We have both another board and MCU in the queue, but it's not ready yet. The board is SOM Vision, MIMXRT1062 or MIMXRT1052 based, where a user confirmed that the Teensy 4.1 build works in principle. The MCU is MIMXRT1176 with a MIMXRT1170_EVK board. The port progressed quite nicely, but there are still hiccups. But including this MCU/board requires changing the nxp-driver and tinyusb lib, and that is the more difficult part.

Reply:
> But including this MCU/board requires changing the nxp-driver and tinyusb lib, and that is the more difficult part

We can do that, as long as it is standard tinyusb, ie just a newer version.